iree/base/api_util.h

```diff
@@ -25,7 +25,6 @@
 namespace iree {
 
 inline iree_status_t ToApiStatus(Status status) {
-  DLOG(ERROR) << status;
```

Comment:
This log line is intentional - it's the only way to get detailed status messages when using the C API.

Reply:
How do we get DLOG ? Should it just be LOG(ERROR) ?

Reply:
I pushed a patch using LOG instead and it seems to work ok. Maybe that is better ?

Reply:
What you had was fine - it was my mistake for forgetting we didn't have this duplicated in the open source repo yet. Really close to getting a public CI setup so we don't do that again :)

Reply:
We merged the PR upstream with your original change, removing the line. It'll take a few minutes for the upstream change to be mirrored and close out the pull request.

We can make a separate change that exposes DLOG (debug only logging macro) and adds the line back in.